UPGRADE_GUIDE.md

```diff
@@ -2,6 +2,13 @@
 
 ## From `0.3.9` to `0.3.10`
 
+### AuthorityLossImminent support removed
+
+The GDK for Unity no longer supports the AuthorityLossImminent authority state. If this feature is currently enabled, it will not trigger the authority state for any reader or writer. The methods to `Ack` the authority state has also been removed.
```

Comment:
```suggestion
The GDK for Unity no longer supports the `AuthorityLossImminent` authority state. If this feature is currently enabled, it will not trigger the authority state for any reader or writer. The methods to `Ack` the authority state has also been removed.
```